app/assets/javascripts/browser-update.js

```diff
@@ -1,11 +1,13 @@
 //browser-update.org notification script, <browser-update.org>
 //Copyright (c) 2007-2009, MIT Style License <browser-update.org/LICENSE.txt>
 
+/* eslint-disable no-var */
+
```

Comment:
Maybe add something like this for future travellers:
```suggestion
/* eslint-disable no-var */
/* This file **must** use `var` to define variables, so that it works in older browsers */

```